Domain/Entities/Abstraction.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Domain.Dto
+﻿namespace Domain.Entities
 {
     public static class Abstraction
```

Comment:
let's remove this class - the enum BetsOptions should be named accordingly to its purpose and move to it's own file

the rest of the consts can be moved where they are used